src/components/OntologyBrowser/index.tsx

```diff
@@ -44,7 +49,44 @@ const desactivateAllSameTerms = (allSameTerms: string[], transferItems: Transfer
     } else return t;
   });
 
-export const updateSqons = (initialSqon: Sqon, value: string[], selectedField: string): Sqon => {
+const opComponent = (fill: string, operation: string) => {
+  switch (operation) {
+    case 'all':
+      return <IntersectionIcon fill={fill} />;
+    case 'some-not-in':
+      return <NotEqualIcon fill={fill} />;
+    default:
+      return <UnionIcon fill={fill} />;
+  }
+};
+
+const OPERATIONS = [
+  {
+    name: 'Any of',
+    component: (fill: string) => opComponent(fill, IN_OP),
```

Comment:
do we really need the opComponent func? Could we just return the desired icon of a given name?

Reply:
this is just to satisfy esLint...